hibianca/FaceReality
# Feature/redesigned layout

# What I did 🧚🏻‍♀️✨

- Redesigned the layout;
- Created start button that goes to the AR View (FR ContentView);
- 3D button changed to the FRContentView;
- Created background to the info button;
- Changed the icon button of the popup that contains the muscles information -> book icon;
- Changed size of the labels on the tab of expressions in the FRContentView;
- Small changes on the buttons.

# How to test

1. Change the your local branch to develop. Before, you must update your local branch.

``` git pull ``` -> to update
``` git checkout develop ``` -> to change your branch

2. Run the app in your device and make sure that 3D and all features are ok.
